src/utils.rs

```diff
@@ -0,0 +1,12 @@
+use regex::Regex;
+
+pub(crate) fn build_regex_search_input(
```

Comment:
I'm kinda new to rust, Can u explain what `pub(crate)` does ?

Reply:
Mean `build_regex_search_input` is public in the `rust_search` only. That means when i can't used it in my crate

Reply:
ohh, Got it.